### DISCUSSION TITLE
Generic tabular data to NDES - modelling cookbook use case

### DISCUSSION THREAD

**Discussion by gwittich** [ternaustralia/ontology_tern#52] [Modelling]
Please include a mapping of:
Generic tabular data to NDES
in the cookbook as a priority.

Gail on Behalf of Gaia Resources.

**Comment by smguru** [comment]
@gwittich can you provide an example dataset?

**Comment by gwittich** [comment]
Hi Guru, I understand we are yet to supply this data. We can leave this on the back burner until we have resolved the DwC points. Is that suitable to you?

**Reply by smguru** [reply]
thanks Gail, that should be good.
